src/test/resources/org/jenkinsci/plugins/reverse_proxy_auth/ReverseProxySecurityRealmTest/secrets/master.key

```diff
@@ -0,0 +1 @@
+624c678c5eeb2425b70ae70248ac87cfb8859cac774e55df34fd22c2eb00d4788292c3009cc4fd2cf0862cb9f27b42ea0c1358da968442658bb4b88dc4dd5e4d8c5420aa643ca1269781d23b4b6e9986fc51bec349110515210f4f2609156f831b4c682057d03e32e7791461f241e27507099e7ad7f593590cc6835486af12cf
```

Comment:
Accidentally added file?

Reply:
No, but needed a `test.key` to avoid a race condition in the test harness.

Reply:
JENKINS-71185